### PR TITLE
Count_Inversion.cpp

### LINKED ISSUE #1664: Count inversion in c++ and java.
The code in the repo are given in c and js. Please assgin me this for c++ and java.

### PR DESCRIPTION
**Pull request by mansikagrawal**
Fixes #1664 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request: 
- Added file Count_Inversion.cpp 

#### Languages Used: 
- CPP 

#### Files Added: 
- Count_Inversion.cpp

### CODE REVIEW

**Comment by mansikagrawal** [comment]
@somya-kapoor  Mam please check I had created a  fresh new commit with the updated file.

File: Count_Inversion/Count_Inversion.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    int aux[n];
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main ()
+{       
+    int n;
+    cin >> n;
+    int arr[n];
+    for(int i=0; i<n; i++)
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Please add space around the operator.

File: Count_Inversion/Count_Inversion.cpp [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    cin >> arr[i];
+    cout << (inversions_count(arr, n))<<endl;
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Please add space around the <<

Comment (mansikagrawal) [reply Count_Inversion/Count_Inversion.cpp]: Space is already present mam

File: Count_Inversion/Count_Inversion.cpp (hunk 1 above)
Comment (mansikagrawal) [reply Count_Inversion/Count_Inversion.cpp]: Space is already present mam.

File: Count_Inversion/Count_Inversion.cpp (hunk 2 above)
Comment (somya-kapoor) [reply Count_Inversion/Count_Inversion.cpp]: <<endl space is not present in it.

File: Count_Inversion/Count_Inversion.cpp (hunk 1 above)
Comment (somya-kapoor) [reply Count_Inversion/Count_Inversion.cpp]: It should be for(int i = 0; i < n; i++)

**Comment by mansikagrawal** [comment]
@somya-kapoor Mam please check now

File: Count_Inversion/Count_Inversion.cpp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    int aux[n];
+    return mergeSort(array, aux, 0, n-1);
+}
+
+int main ()
+{       
+    int n;
+    cin >> n;
+    int arr[n];
+    for(int i=0; i < n; i++)
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.cpp]: Add space around = also

**Comment by mansikagrawal** [comment]
@somya-kapoor  Please check again mam

**Comment by somya-kapoor** [comment]
@mansikagrawal Good work. Looks good to me. Just squash the commits, then it will be ready to merge.

**Comment by somya-kapoor** [comment]
@mansikagrawal Are you working on it?

**Comment by mansikagrawal** [comment]
@somya-kapoor Mam I'll squash the commit today
Sorry for being late

**Pull request closed without merging**